src/rgw/rgw_sal_motr.cc

```diff
@@ -491,6 +491,15 @@ int MotrUser::remove_user(const DoutPrefixProvider* dpp, optional_yield y)
       }
     }
   }
+
+  //Delete email id 
+  if (!info.user_email.empty()) {
+    rc = store->do_idx_op_by_name(RGW_IAM_MOTR_EMAIL_KEY,
+		             M0_IC_DEL, info.user_email, bl);
+    if (rc < 0) {
+       ldpp_dout(dpp, 0) << "Unable to delete email id or email not exist" << rc << dendl;
+    }
```

Comment:
This will show up if we delete a user who does not have email address which is not an error. We should skip that case while printing this error.
```
if (rc < 0 && rc != -ENOENT) {
    ldpp_dout(dpp, 0) << "Unable to delete email id" << rc << dendl;
}